server/src/test/java/org/elasticsearch/client/transport/TransportClientIT.java

```diff
@@ -66,7 +66,6 @@ public void testNodeVersionIsUpdated() throws IOException, NodeValidationExcepti
                 .put("transport.type", getTestTransportType())
                 .put(Node.NODE_DATA_SETTING.getKey(), false)
                 .put("cluster.name", "foobar")
-                .put(TestZenDiscovery.USE_ZEN2.getKey(), true)
```

Comment:
This (and some of the other occurrences removed in this PR) should be set to Zen1 or Zen2 depending on whether the surrounding test uses the corresponding disco type. If we want to randomize the tests to run with Zen1 / Zen2 for a while, it's probably better to add a field `useZen2` to ESTestCase (similar to `useNio`) that will for now be set to Zen2, but could also be set to randomBoolean() for a while. This line here should then be changed to `.put(TestZenDiscovery.USE_ZEN2.getKey(), getUseZen2())`